app/router.js

```diff
@@ -95,6 +95,7 @@ router.map(function() {
   });
   this.route('my-sessions', function() {
     this.route('list', { path: '/:session_status' });
+    this.route('view', { path: '/:session_id' });
```

Comment:
@ritikamotwani make the path as `/s/:session_id` 
The acceptance tests are failing because this definition of `list` and `view` routes is ambiguous. It will consider `past` and `upcoming` as ids of sessions, which don't exist :)